molgenis-navigator/src/main/frontend/test/unit/specs/store/actions.spec.js

```diff
@@ -0,0 +1,330 @@
+import api from '@molgenis/molgenis-api-client'
+import td from 'testdouble'
+import actions, { GET_ENTITIES_IN_PACKAGE } from 'src/store/actions'
+import { RESET_PATH, SET_ENTITIES, SET_ERROR, SET_PACKAGES, SET_PATH } from 'src/store/mutations'
+import testAction from '../../utils/action.utils'
+
+describe('actions', () => {
+  afterEach(() => td.reset())
+
+  describe('QUERY_PACKAGES', function () {
+    it('should fetch the packages and call the SET_PACKAGES mutation', done => {
+      const package1 = {id: 'pack1', label: 'packLabel1'}
+      const response = {
+        items: [package1]
+      }
+
+      const get = td.function('api.get')
+      td.when(get('/api/v2/sys_md_Package?sort=label&num=1000')).thenResolve(response)
+      td.replace(api, 'get', get)
+
+      testAction(actions.__QUERY_PACKAGES__, null, {}, [{type: SET_PACKAGES, payload: response.items}], [], done)
```

Comment:
Having a method call with 6 unnamed params seems like a bad idea to me, the fact that half of them are null or empty also raises some flags with me. I think it makes the test hard to read. calling the action in the same way as the component code does seem clearer to me. 

If you do want to test using the testAction setup is would suggest naming the params ( using a object or vars) and/or creating specific named functions that have sensible defaults set.